Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -3676,11 +3689,20 @@ private ObservableList<ElementInfo> GetAllElementsFromPage(string path, List<eEl
 
                         //get Element Type
                         Tuple<string, eElementType> elementTypeEnum = GetElementTypeEnum(htmlNode: htmlElemNode);
+                        bool isFrameElement = elementTypeEnum.Item2 == eElementType.Iframe;
```

Comment:
simplify code by not checking anything specific to IFram on filtering, just do learning if item is selected to be learned,